packages/gatsby-cli/src/create-cli.ts

```diff
@@ -534,6 +508,95 @@ Creating a plugin:
         report.log(`Telemetry collection ${enabled ? `enabled` : `disabled`}`)
       }),
     })
+    .command({
+      command: `plugin [cmd]`,
+      describe: `Useful commands relating to Gatsby plugins`,
+      builder: yargs =>
+        yargs.positional(`cmd`, {
+          choices: [`docs`],
+          describe: "Valid commands include `docs`.",
+          type: `string`,
+        }),
+      handler: handlerP(({ cmd }) => {
+        if (cmd === `docs`) {
+          console.log(`
+      Using a plugin:
+      - What is a Plugin? (https://www.gatsbyjs.com/docs/what-is-a-plugin/)
+      - Using a Plugin in Your Site (https://www.gatsbyjs.com/docs/using-a-plugin-in-your-site/)
+      - What You Don't Need Plugins For (https://www.gatsbyjs.com/docs/what-you-dont-need-plugins-for/)
+      - Loading Plugins from Your Local Plugins Folder (https://www.gatsbyjs.com/docs/loading-plugins-from-your-local-plugins-folder/)
+      - Plugin Library (https://www.gatsbyjs.com/plugins/)
+
+      Creating a plugin:
+      - Naming a Plugin (https://www.gatsbyjs.com/docs/naming-a-plugin/)
+      - Files Gatsby Looks for in a Plugin (https://www.gatsbyjs.com/docs/files-gatsby-looks-for-in-a-plugin/)
+      - Creating a Generic Plugin (https://www.gatsbyjs.com/docs/creating-a-generic-plugin/)
+      - Creating a Local Plugin (https://www.gatsbyjs.com/docs/creating-a-local-plugin/)
+      - Creating a Source Plugin (https://www.gatsbyjs.com/docs/creating-a-source-plugin/)
+      - Creating a Transformer Plugin (https://www.gatsbyjs.com/docs/creating-a-transformer-plugin/)
+      - Submit to Plugin Library (https://www.gatsbyjs.com/contributing/submit-to-plugin-library/)
+      - Source Plugin Tutorial (https://www.gatsbyjs.com/tutorial/source-plugin-tutorial/)
+      - Maintaining a Plugin (https://www.gatsbyjs.com/docs/maintaining-a-plugin/)
+      - Join Discord #plugin-authoring channel to ask questions! (https://gatsby.dev/discord/)
+                 `)
+        } else {
+          console.log(`Current valid subcommands are: 'docs'`) // right now this is hardcoded because we only have a single command
+        }
+      }),
+    })
+    .command({
+      command: `options [cmd] [key] [value]`,
+      describe: `View or set your gatsby-cli configuration settings.`,
```

Comment:
I think that "view" with `gatsby options` and set with `gatsby options set` could be seperate commands (if yargs allow it that is). From the description (and the screenshot that @gillkyle provided in https://github.com/gatsbyjs/gatsby/pull/26856#pullrequestreview-488033386 ) it doesn't seem straight forward how to actually view options.

This is also somewhat similar in descriptions of positional arguments - i.e. `[cmd]` talks about package manager but this argument is not really about that.